src/css/components/_slider.scss

```diff
@@ -0,0 +1,92 @@
+.sliderOverlay {
+    position          : fixed;
+    top               : 0;
+    left              : 0;
+    right             : 0;
+    bottom            : 0;
+    background-color   : rgba(0, 0, 0, 0.75);
+    width: $master-width;
+    height: $master-height;
+    margin: 25px;
+    z-index:1001;
+}
+
+.sliderModal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    margin-right: -50%;
```

Comment:
Looks like this attribute may be unnecessary

Reply:
Removed in https://github.com/smartdevicelink/generic_hmi/pull/393/commits/753e34795d8550eebc85d23480f63dbf9b190ef2